swagger.yml

```diff
@@ -12,124 +12,137 @@ info:
+        - "application/json"
       parameters:
-      - in: "body"
-        name: "body"
-        description: "Login parameters"
-        required: true
-        schema:
-          $ref: "#/definitions/login"
+        - in: "body"
+          name: "body"
+          description: "Login parameters"
+          required: true
+          schema:
+            $ref: "#/definitions/login"
       responses:
         "200":
-          description: "Successful Login"   
+          description: "Successful Login"
         "401":
-          $ref: '#/responses/UnauthenticatedError'
+          $ref: "#/responses/UnauthenticatedError"
         "403":
-          $ref: '#/responses/UnauthorizedError'
+          $ref: "#/responses/UnauthorizedError"
         "405":
           description: "Method not allowed"
         "503":
-          $ref: '#/responses/InternalServerError'
+          $ref: "#/responses/InternalServerError"
       security:
-      - basicAuth: []
+        - basicAuth: []
   /register:
     post:
       tags:
-      - "register"
+        - "register"
       summary: "Create user"
       description: "This end point is to register a user"
       operationId: "registerUser"
       produces:
-      - "application/json"
+        - "application/json"
       parameters:
-      - in: "body"
-        name: "body"
-        description: "Created user object"
-        required: true
-        schema:
-          $ref: "#/definitions/Register"
+        - in: "body"
+          name: "body"
+          description: "Created user object"
+          required: true
+          schema:
+            $ref: "#/definitions/Register"
       responses:
         "201":
-          description: "User created successfully"        
+          description: "User created successfully"
         "400":
           description: "Bad request"
         "405":
           description: "Method not allowed"
         "503":
-          $ref: '#/responses/InternalServerError'
+          $ref: "#/responses/InternalServerError"
   /user:
     get:
       tags:
-      - "user"
+        - "user"
       summary: "Get the details of the users registered in the system"
       description: "This can be done only by an authenticated user"
       operationId: "user"
       produces:
-      - "application/json"
+        - "application/json"
       parameters:
-      - name: "username"
-        in: "query"
-        description: "The user name for login"
-        required: true
-        type: "string"
-      - name: "password"
-        in: "query"
-        description: "The password for login in clear text"
-        required: true
-        type: "string"
+        # - name: "username"
+        #   in: "query"
+        #   description: "The user name for login"
+        #   required: true
+        #   type: "string"
+        # - name: "password"
+        #   in: "query"
+        #   description: "The password for login in clear text"
+        #   required: true
+        #   type: "string"
```

Comment:
Dont keep commented values in pull requests. Else we will need a pull request again just to remove them later

Reply:
resolved !!